src/pyramid_jwt/__init__.py

```diff
@@ -9,7 +9,7 @@ def includeme(config):
 
 
 def create_jwt_authentication_policy(config, private_key=None, public_key=None,
-        algorithm=None, expiration=None, leeway=None,
+        algorithm=None, expiration=None, leeway=None, audience=None,
```

Comment:
I am a bit worried about adding a parameter in the middle here: theoretically you break backwards compatibility of someone does not use named parameters.